_analyzers/token-filters/predicate-token-filter.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Predicate token filter
+parent: Token filters
+nav_order: 340
+---
+
+# Predicate token filter
+
```

Comment:
Line 10, second sentence: "in the analysis predicate context" => "in the context of the analysis predicate"?

Reply:
Analysis predicate context is the proper term